larq/metrics.py

```diff
@@ -0,0 +1,99 @@
+import tensorflow as tf
+from larq import utils
+import numpy as np
+
+
+class MeanChangedValues(tf.keras.metrics.Mean):
```

Comment:
@koenhelwegen @jamescook106 What do you think about the name, I am not sure if it is the best we can come up with.

Reply:
I like `FlipRatio`. Other options would be `ChangeRatio` or `AlteringRatio`.

Reply:
Changed it to `FlipRatio`